src/components/HeadersCustom.js

```diff
@@ -0,0 +1,35 @@
+import Head from 'next/head';
+import React from 'react';
+
+const HeadersCustom = ({
+   title = 'AngelFolio',
+   description = "Angel's portfolio site and blog. Come check it out!",
+}) => {
+   return (
+      <Head>
+         <title>AngelFolio</title>
+         <meta name='title' content={title} />
```

Comment:
[See Here for the Next.JS metadata export stuff that this sort of maps to](https://nextjs.org/docs/app/api-reference/functions/generate-metadata#basic-fields)

Reply:
On second thought I'd have to refactor most of my pages (most of which have some degree of reactivity so they are marked with "use client") so imo it might not be worth the time right now to change all uses of custom header to use `export const metadata` or `generateMetadata`